applications/launchpad/gui-react/src/components/Backdrop/index.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components'
 
 const Backdrop = styled.div<{ opacity?: number; borderRadius?: string }>`
```

Comment:
NIT: I would suggest to add `$` to prop name so it won't be added to the DOM
```suggestion
const Backdrop = styled.div<{ opacity?: number; $borderRadius?: string }>`
```

Reply:
Actually, we can do this also for the `opacity`: `$opacity?: number`.

Reply:
I'll get it on the next PR.